src/main/java/de/hysky/skyblocker/skyblock/entity/MobGlow.java

```diff
@@ -73,6 +80,13 @@ public static boolean isStarred(Entity entity) {
 		List<ArmorStandEntity> armorStands = getArmorStands(entity);
 		return !armorStands.isEmpty() && armorStands.getFirst().getName().getString().contains("✯");
 	}
+	public static String  getArmourStandName(Entity entity) {
```

Comment:
Uh... not the british/canadian/australian/whatever english. Can you change it to `armor` since the rest of the code is `armor`. Also can you run a code cleanup as there are some messed up formatting. (`Command/Control` + `Option` + `L`).

Also, I think it would be nicer to return an empty string instead of null, or annotate the method with `@Nullable`. It would also be nice if you add javadocs.